src/librustc_metadata/encoder.rs

```diff
@@ -869,6 +871,13 @@ impl<'a, 'b: 'a, 'tcx: 'b> IsolatedEncoder<'a, 'b, 'tcx> {
         !self.tcx.sess.opts.output_types.should_trans()
     }
 
+    fn const_qualif(&self, mir: u8, body_id: hir::BodyId) -> ConstQualif {
```

Comment:
This should ideally call `mir_const_qualif` itself instead of taking the value as an argument.